perllib/FYR/Queue.pm

```diff
@@ -932,18 +935,20 @@ sub make_representative_email ($$) {
             );
 
     my $headers = {
-        From => [ $msg->{sender_email}, $msg->{sender_name} ],
-        To => [ [ $msg->{recipient_email}, $msg->{recipient_name} ] ],
+        From => mySociety::Email::format_email_address($msg->{sender_name}, $msg->{sender_email}),
+        To => mySociety::Email::format_email_address($msg->{recipient_name}, $msg->{recipient_email}),
         Subject => $subject,
         Date => strftime('%a, %e %b %Y %H:%M:%S %z', localtime(FYR::DB::Time())),
         'Message-ID' => email_message_id($msg->{id}),
-        _body_ => $bodytext
     };
     if (test_dmarc($msg->{sender_email})) {
-        $headers->{From} = [ $sender, $msg->{sender_name} ];
-        $headers->{'Reply-To'} = [ [ $msg->{sender_email}, $msg->{sender_name} ] ],
+        $headers->{From} = mySociety::Email::format_email_address($msg->{sender_name}, $sender);
+        $headers->{'Reply-To'} = mySociety::Email::format_email_address($msg->{sender_name}, $msg->{sender_email});
     }
-    return mySociety::Email::construct_email($headers);
+    return Email::MIME->create(
+        header_str => [%$headers],
+        parts => [ $bodytext ]
+    )->as_string;
```

Comment:
I think this is ok as it's set on the individual parts so I don't need it on the overall email.